doc/manuals/user/walkthrough_apiv2.md

```diff
@@ -788,6 +791,9 @@ notifications due to throttling):
     clever enough to know that the value previous to the updateContext
     request was also 765 so no actual update has occurred and
     consequently no notification is sent.
+-   update Room1 pressure to 765 with forcedUpdate URI option: In this 
+    case broker will send the notification, because of the forcedUpdate 
```

Comment:
forcedUpdate -> `forcedUpdate`

(also in the next line)

Reply:
fixed in bb8e9ae